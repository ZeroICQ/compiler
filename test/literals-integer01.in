10 20
300 4000
50000
$7f $7ff
$efffff
%010101 %00001 %1 %0

