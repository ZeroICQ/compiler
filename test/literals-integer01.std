(1:0)	integer literal	[10]	"10"
(1:3)	integer literal	[20]	"20"
(2:0)	integer literal	[300]	"300"
(2:4)	integer literal	[4000]	"4000"
(3:0)	integer literal	[50000]	"50000"
(4:0)	integer literal	[127]	"$7f"
(4:4)	integer literal	[2047]	"$7ff"
(5:0)	integer literal	[15728639]	"$efffff"
(6:0)	integer literal	[21]	"%010101"
(6:8)	integer literal	[1]	"%00001"
(6:15)	integer literal	[1]	"%1"
(6:18)	integer literal	[0]	"%0"
(6:20)	EOF	[end of file]	"EOF"
