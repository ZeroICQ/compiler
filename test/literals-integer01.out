(1:0)	integer literal	[10]	"10"
(1:3)	integer literal	[20]	"20"
(2:0)	integer literal	[300]	"300"
(2:4)	integer literal	[4000]	"4000"
(3:0)	integer literal	[50000]	"50000"
(4:0)	hex integer literal	[127]	"$7f"
(4:4)	hex integer literal	[2047]	"$7ff"
(5:0)	hex integer literal	[15728639]	"$efffff"
(6:0)	bin integer literal	[21]	"%010101"
(6:8)	bin integer literal	[1]	"%00001"
(6:15)	bin integer literal	[1]	"%1"
(6:18)	bin integer literal	[0]	"%0"
(6:20)	EOF	[]	" "
